go.mod

```diff
@@ -22,8 +22,8 @@ require (
 	github.com/miekg/dns v1.1.61
 	github.com/miekg/pkcs11 v1.1.1
 	github.com/nxadm/tail v1.4.11
-	github.com/prometheus/client_golang v1.15.1
-	github.com/prometheus/client_model v0.4.0
+	github.com/prometheus/client_golang v1.20.5
```

Comment:
If we jump straight to [prometheus/client_golang v1.22.0](https://github.com/prometheus/client_golang/releases/tag/v1.22.0), we can pull in https://github.com/prometheus/client_golang/pull/1765, which lets us avoid the big zstd dependency.

Reply:
Good call, done.